how-to/4--self-host-workspace/.gitignore

```diff
@@ -0,0 +1,2 @@
+hosted-workspace.zip
+public
```

Comment:
We should put some flavor of formatting rules `eslint / prettier / vscode / editorconfig` (whatever we like to use) into the root of the repo so all of these examples have some baseline consistency, like new lines at the ends of files.

It creates better diffs in pull requests and the consistency feels like higher quality

Reply:
https://appoji.jira.com/browse/PE-198

Reply:
Agreed, I can address this in a separate PR if you would like. Just want to keep this PR semi focused.

Reply:
awesome sounds good